dropwizard-db/src/test/java/io/dropwizard/db/DataSourceConfigurationTest.java

```diff
@@ -107,6 +109,17 @@ public void testInlineUserPasswordConfiguration() throws Exception {
         assertThat(ds.getUser()).isNull();
         assertThat(ds.getPassword()).isNull();
     }
+    // Test added for https://github.com/dropwizard/dropwizard/issues/1037
+    @Test
+    public void testInitialSizeZeroIsAllowed() throws Exception {
+        try {
+            DataSourceFactory ds = getDataSourceFactory("yaml/empty_initial_pool.yml");
```

Comment:
Surrounding the statement with try-catch doesn't gain anything. The test will fail anyway, if any uncaught exception occurs.